lib/matplotlib/texmanager.py

```diff
@@ -176,14 +175,14 @@ def get_basefile(cls, tex, fontsize, dpi=None):
             src.encode('utf-8'),
             usedforsecurity=False
         ).hexdigest()
-        filepath = Path(cls._texcache)
+        filepath = cls._texcache
 
         num_letters, num_levels = 2, 2
         for i in range(0, num_letters*num_levels, num_letters):
             filepath = filepath / Path(filehash[i:i+2])
```

Comment:
```suggestion
            filepath = filepath / filehash[i:i+2]
```
Should work without.

Reply:
sure